lib/Workflow/GlobalAction.php

```diff
@@ -37,11 +37,15 @@ class GlobalAction implements NotesAwareInterface
      */
     private $expressionService;
 
-    public function __construct(string $name, array $options, ExpressionService $expressionService)
+    /** @var string */
+    private $workflowName;
+
+    public function __construct(string $name, array $options, ExpressionService $expressionService, string $workflowName)
```

Comment:
creation of new global actions [here](https://github.com/pimcore/pimcore/blob/75ee283823dcb5dd008d7a41a43153d204d64174/lib/Workflow/Manager.php#L98) also needs to set the workflow name